src/components/HURUmap/ProfileKeyMetric/index.js

```diff
@@ -0,0 +1,58 @@
+import { Typography, LinearProgress } from "@material-ui/core";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const ProfileKeyMetric = ({ percentage, description, color, summary }) => {
```

Comment:
```suggestion
const KeyMetric = ({ formatedValue, title, color, description }) => {
```

As we discussed in the call:

1. `KeyMetric` is part of `SubCategory` and not `Profile`.
2. We should default to using `title`, `value`, and `description` in all our components,
3. `formatedValue` if available should be used when we need to show `value` with formatting e.g. percentage, currency, etc.

Reply:
Quick question on the `formattedValues` - Do we get both `formattedValue` and `value` and does the formattedValue come with it's sign i.e % or £ etc? @kilemensi

Reply:
Yes and Yes @gertie-sheshe

Reply:
Fixed